src/schema/rules/datatypes/tabular_metadata.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- suffixes:
+  - sessions
+  extensions:
+  - .tsv
+  - .json
+  entities:
+    subject: required
+    session: optional
```

Comment:
Just wanted to note that session actually should be prohibited for sessions files. I originally had `session: required` in #985 before I realized this.